src/lib.rs

```diff
@@ -9,7 +9,7 @@
 //! [`compiletest_rs`](https://crates.io/crates/compiletest_rs), looks as follows:
 //!
 //! ```rust,ignore
-//! use std::{fs::read_to_string, path::PathBuf, process::Command};
+//! use std::{env, fs::read_to_string, path::PathBuf, process::Command};
```

Comment:
Is there a reason it's marked `ignore`? We could have caught this earlier?

Reply:
I'm not sure I want my doc tests spinning up arbitrary commands. But maybe I'm over-thinking it!

Reply:
I'll merge this as-is, and you can have a think about it.